packages/@vuepress/theme-default/src/client/styles/vars.scss

```diff
@@ -65,6 +65,20 @@
   --sidebar-width-mobile: calc(var(--sidebar-width) * 0.82);
   --content-width: 740px;
   --homepage-width: 960px;
+
+  // transition vars
+  --t-duration: 0.3s;
+  --t-duration-slide: 0.3s;
+  --t-timing-function: ease;
```

Comment:
Maybe we could merge them into a single var, because "duration" and "timing function" always appear together in following usage.

Reply:
822d845

OK. Users can control `transition-duration`, `transition-timing-function` and `transition-delay` in a single variable in this way.